app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: None, cafe: None) -> None:
```

Comment:
The function parameters should have type annotations instead of using 'None'. Consider using appropriate types for 'friends' and 'cafe'.